alma/alma-installments-prestashop
# [cart-item] Integrate the block see the insurance available for the

### Reason for change



[Linear task](https://linear.app/almapay/issue/ECOM-1598/[🧩-ps]-integrate-the-block-see-the-insurance-available-for-the-item)

### Code changes

Add block to display insurance modal clicking on the button in the cart item without insurance


### How to test

Add two same product with one insurance and see the block insurance in the item cart without insurance
_As a reviewer, you are encouraged to test the PR locally._



### Checklist for authors and reviewers




### Non applicable
